cmd/version.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
```

Comment:
Add a new line between `fmt` and `github.com/spf13/cobra` to be consistent.

Reply:
@tompizmor if you don't have it you should probably add formatting on save in goland

Reply:
Yeah! I just added it back. I would swear I already configured it a FileWatcher to run gofmt but it wasn't there 🤷‍♂️